src/pages/staticPages/workInstructions/WorkInstructionsMain.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import MainHeading from '../../../common/components/mainHeading/MainHeading';
+import { Trans, useTranslation } from 'react-i18next';
+import { useBreadcrumbs } from './WorkInstructionsPage';
+import { BREADCRUMBS } from './Breadcrumbs';
+import { Link } from 'hds-react';
+import styles from './WorkInstructions.module.scss';
+
+const WorkInstructionsMain: React.FC = () => {
+  const { t } = useTranslation();
+  const { setBreadcrumbs } = useBreadcrumbs();
+
+  useEffect(() => {
+    const updateBreadcrumbs = () => setBreadcrumbs([BREADCRUMBS.tyoOhjeet]);
+
+    updateBreadcrumbs();
+  }, [setBreadcrumbs]);
+  return (
+    <>
+      <MainHeading spacingBottom="xl">{t('workInstructions:main:header')}</MainHeading>
+
+      <div className={styles.content}>
+        <Trans
+          i18nKey="workInstructions:main:content"
+          components={{
+            p: <p />,
+            a: (
+              <Link
+                href={t('workInstructions:sideNav:externalLinks:permitsAndInstructions:url')}
+                external
+                openInNewTab
+                openInNewTabAriaLabel="components:link:openInNewTabAriaLabel"
+                openInExternalDomainAriaLabel={t('components:link:openInExternalDomainAriaLabel')}
```

Comment:
`common` missing from the translation path here also.

Same thing seems to be with all Links openInNewTabAriaLabel and openInExternalDomainAriaLabel props.